recipes-browser/wpewebkit/wpewebkit_2.38.0.bb

```diff
@@ -0,0 +1,22 @@
+require wpewebkit.inc
+require conf/include/devupstream.inc
+
+FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}:"
+
+SRC_URI = "https://wpewebkit.org/releases/${BPN}-${PV}.tar.xz;name=tarball \
+           "
+
+SRC_URI[tarball.sha256sum] = "e653e12eca63e6371c06d4cd9efcd5492db67f7d9ef324fd5d8668b1151e72b2"
+
+DEPENDS += " libwpe"
+RCONFLICTS:${PN} = "libwpe (< 1.10) wpebackend-fdo (< 1.10)"
+
+SRC_URI:class-devupstream = "\
+    git://github.com/WebKit/WebKit.git;protocol=https;branch=main \
+"
+# main branch
+SRCREV:class-devupstream = "main"
```

Comment:
wrong syntax. It should be ${AUTOREV}.


In addition, I prefer to keep pointing to the specific reference where the release (2.38, in this case) was forked:

```
# WPE 2.38.X branch was forked from the main branch in this commit
SRCREV:class-devupstream = "... (check hash from the upstream repo)"
```